tests/system_tests/test_blueapi_system.py

```diff
@@ -51,13 +51,18 @@
 To enable and execute these tests, set `REQUIRES_AUTH=1` and provide valid credentials.
 """
 
-# Step 1: Ensure a message bus that supports stomp is running and available:
-#   src/script/start_rabbitmq.sh
+# Start devices
+#   1. clone https://github.com/epics-containers/example-services
+#   2. cd example-services
+#   2. docker compose up bl01t-di-cam-01 bl01t-mo-sim-01 ca-gateway
 #
-# Step 2: Start the BlueAPI server with valid configuration:
-#   blueapi -c tests/system_tests/config.yaml serve
+# Start services
```

Comment:
Can it be made a bit cleared that the rest of these commands are back in blueapi not still in example-services from above

Reply:
How's the new wording/arrangment?

Reply:
much better, thanks